lib/pbench/server/api/resources/users_api.py

```diff
@@ -59,15 +59,22 @@ def post(self):
             abort(HTTPStatus.INTERNAL_SERVER_ERROR, message="INTERNAL ERROR")
 
         if not user:
-            self.logger.exception("No such user exists with the given email ID")
-            abort(HTTPStatus.BAD_REQUEST, message="No such user exists with the given email ID")
+            self.logger.warning("No such user exists with the given email ID")
```

Comment:
Will we have enough context when we look at the log to know that this message comes from a "forgot password" request?  If not, you should include that in the text.  (Ditto, perhaps, for the other `logger` calls.)